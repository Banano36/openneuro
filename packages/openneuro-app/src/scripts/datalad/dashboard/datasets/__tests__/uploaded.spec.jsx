import React from 'react'
import { shallow } from 'enzyme'
import Uploaded from '../uploaded.jsx'

const defProps = {
  uploader: {
    name: 'A User',
  },
  created: null, // Avoid testing the current date
}

describe('dashboard/datasets/Uploaded', () => {
  it('renders successfully', () => {
    const wrapper = shallow(<Uploaded {...defProps} />)
    expect(wrapper).toMatchSnapshot()
  })
})
