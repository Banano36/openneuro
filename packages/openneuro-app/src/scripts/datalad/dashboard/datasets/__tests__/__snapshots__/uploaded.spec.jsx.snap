// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dashboard/datasets/Uploaded renders successfully 1`] = `
<div
  className="meta-container"
>
  <p
    className="date"
  >
    uploaded 
    by 
    <span
      className="name"
    >
      A User
    </span>
     on
     
    <span
      className="time-ago"
    >
       - 
       ago
    </span>
  </p>
</div>
`;
